rdt/hyper_transformer.py

```diff
@@ -188,8 +193,8 @@ def get_config(self):
                 - transformers: A dictionary mapping column names to their transformer instances.
         """
         return {
-            'sdtypes': self.field_sdtypes,
-            'transformers': self.field_transformers
+            'sdtypes': {**self.provided_field_sdtypes, **self._fitted_field_sdtypes},
```

Comment:
I think we may want to have a method that correctly combines these two dictionaries into one. Then anywhere that needs to loop over the whole dict can just call that method. We should also probably have a method for setting things in that dictionary that removes from the `_fitted` one if necessary. So something like:
```
_get_field_sdtypes() => returns combination of both dicts
_set_field_sdtype() => adds to provided dict and removes from _fitted
```

Either that or we could make a class to manage all of this but that may be overkill.